src/pageHead.test.jsx

```diff
@@ -0,0 +1,115 @@
+	PAGE_TITLE_CLASS,
+	PAGE_ACTIONS_CLASS,
+} from './PageHead';
+import PageActionButton from './PageActionButton';
+import Popover from './Popover';
+
+const menu = [
+		<PageActionButton icon='search' label='Search' />,
+		<PageActionButton icon='edit' label='Edit' />,
+		<Popover
+			align='left'
+			trigger={
+				<PageActionButton icon='overflow' label='More' />
+			}
+			menuItems={[
+				<Link to='somepath/'>First option</Link>,
+				<Link to='somepath/'>Second option</Link>,
+				<Link to='somepath/'>Third option</Link>,
+			]}
+		/>
+	],
+	pageTitle = 'Page title',
+	subtitle = 'Sub title';
+
+let pageHead,
+	pageHeadEl;
+
+describe('PageHead', function() {
+	beforeEach(() => {
+		pageHead = TestUtils.renderIntoDocument(
+			<PageHead title={pageTitle} />
+		);
+		pageHeadEl = ReactDOM.findDOMNode(pageHead);
+	});
+
+	afterEach(() => {
+		pageHead = null;
+		pageHeadEl = null;
+	});
+
+	it('exists', function() {
+		expect(pageHeadEl).not.toBeNull();
+	});
+
+	describe('title', () => {
+		it('should display a title area', () => {
+			expect(pageHeadEl.innerHTML.indexOf(pageTitle)).not.toBe(-1);
+		});
+		it(`should have a 'Flex' component with a class of ${PAGE_TITLE_CLASS}`, () => {
+			const flexItems = findComponentsWithType(pageHead, 'Flex');
+			const pageTitle = flexItems.filter(flex => flex.props.className.indexOf(PAGE_TITLE_CLASS) !== -1);
+			expect(pageTitle.length).toBe(1);
+		});
+		it(`should NOT have a 'Flex' component with a class of ${PAGE_ACTIONS_CLASS}`, () => {
+			const flexItems = findComponentsWithType(pageHead, 'Flex');
+			const pageActions = flexItems.filter(flex => flex.props.className.indexOf(PAGE_ACTIONS_CLASS) !== -1);
+			expect(pageActions.length).toBe(0);
+		});
+		it('should NOT have a \'<p>\' tag', () => {
+			const subtitleEl = pageHeadEl.getElementsByTagName('P');
+			expect(subtitleEl.length).toBe(0);
+		});
+	});
+
+	describe('subtitle', () => {
+		beforeEach(() => {
+			pageHead = TestUtils.renderIntoDocument(
+				<PageHead title={pageTitle} subtitle={subtitle} />
+			);
+			pageHeadEl = ReactDOM.findDOMNode(pageHead);
+		});
+
+		it('should display a sub title area', () => {
+			expect(pageHeadEl.innerHTML.indexOf(subtitle)).not.toBe(-1);
+		});
+
+		it('should have a \'<p>\' tag with a class of \'text--secondary\'', () => {
+			const subtitleEl = pageHeadEl.getElementsByTagName('P');
+			expect(subtitleEl.length).toBe(1);
+			expect(subtitleEl[0].classList).toContain('text--secondary');
+		});
+	});
+
+	describe('menuItems', () => {
+		beforeEach(() => {
+			pageHead = TestUtils.renderIntoDocument(
+				<PageHead title={pageTitle} menuItems={menu} />
+			);
+			pageHeadEl = ReactDOM.findDOMNode(pageHead);
+		});
+
+		it(`should have a 'Flex' component with a class of ${PAGE_ACTIONS_CLASS}`, () => {
+			const flexItems = findComponentsWithType(pageHead, 'Flex');
+			const pageActions = flexItems.filter(flex => flex.props.className && flex.props.className.indexOf(PAGE_ACTIONS_CLASS) !== -1);
```

Comment:
you can probably use [`scryRenderedDOMComponentsWithClass()`](https://facebook.github.io/react/docs/test-utils.html#scryrendereddomcomponentswithclass) to select all the `PAGE_ACTIONS_CLASS` components regardless of 'type' - as long as you have a `PAGE_ACTIONS_CLASS` component, it probably doesn't matter if it's a `<Flex>`